pnpm-global/5/pnpm-lock.yaml

```diff
@@ -0,0 +1,7 @@
+lockfileVersion: 5.3
```

Comment:
What is this for?

Reply:
hmmm. Not sure. Must have been an old `pnpm link` i'll get rid of it.